spec/features/saml/sp_initiated_slo_spec.rb

```diff
@@ -213,4 +213,34 @@
       expect(current_path).to eq('/')
     end
   end
+
+  context 'logged out of IDP' do
+    let(:user) { create(:user, :signed_up) }
+
+    context 'signed into one SP' do
+      before do
+        sign_in_and_2fa_user(user)
+        visit sp1_authnrequest
+
+        request = OneLogin::RubySaml::Logoutrequest.new
```

Comment:
This threw me off at first because it looked like you were making a sign out request from the SP before the user was logged out of the IdP. What do you think about moving this line to line 229, right before `visit request.create(settings)`?